core/jazz-ui/src/app/pages/environment-codequality/env-codequality-section.component.scss

```diff
@@ -144,6 +145,14 @@
     &:last-child {
       margin-right: 0;
     }
+
+    .metrics-card-content {
```

Comment:
Are we re-purposing the styles for metrics in code quality? Given they are two different components, perhaps the styles should be specific to them, no?

Reply:
.metric-card refers to the tiles that go under the code quality graph since the code quality query returns {data: {metrics:[]}}. These styles will likely be copied into the metrics component and will sit as duplicates until I can migrate to a unified styling convention.